src/cljs/athens/events.cljs

```diff
@@ -486,9 +486,11 @@
                        db/get-block)
         new-block {:db/id (:db/id block) :block/order (count (:block/children older-sib))}
         reindex-blocks (->> (dec-after (:db/id parent) (:block/order block)))]
-    {:transact! [[:db/retract (:db/id parent) :block/children (:db/id block)]
-                 {:db/id (:db/id older-sib) :block/children [new-block]} ;; becomes child of older sibling block — same parent but order-1
-                 {:db/id (:db/id parent) :block/children reindex-blocks}]}))
+    (if (zero? (:block/order block)) ; If the block is already the top in it's level, do not indent
+      {}
```

Comment:
Even if it's not necessary anymore, not sure how I feel about empty `{}` vs `nil` in general for return value. Thoughts?

Reply:
The `{}` was to indicate explicitly that `no-op` was taking place. I think it has some semantic clarity.

Reply:
I specifically read somewhere yesterday in one of our libraries' official docs that this was an acceptable practice. Maybe it was in the `re-frame` docs? I will see if I can find it... 😅

Reply:
Lemme know if you find it @nthd3gr33!